.gitignore

```
@@ -0,0 +1,4 @@
+moz_preprocess/__pycache__/__init__.cpython-310.pyc
+moz_preprocess/__pycache__/bq_utils.cpython-310.pyc
+moz_preprocess/__pycache__/preprocess.cpython-310.pyc
```

I think you can probably make the following change so that git will [ignore any files](https://www.freecodecamp.org/news/gitignore-file-how-to-ignore-files-and-folders-in-git/) in the `__pycache__` directory, as well as the directory itself:
```suggestion
moz_preprocess/__pycache__/
```